### PR TITLE
Use local assets

### PR DESCRIPTION
**Pull request by klaseskilson**
Again, thanks for a nice gem.

This PR includes Leaflet locally instead of using a CDN.

### CODE REVIEW

**Comment by klaseskilson** [comment]
I believe this is the reason for the test failures: https://github.com/axyjo/leaflet-rails/issues/65

**Comment by pedantic-git** [comment]
Makes sense! I've submitted https://github.com/axyjo/leaflet-rails/pull/66 to address that.

Over here, can you please add the following just below the first line of `spec_helper.rb`?

```ruby
# Monkeypatch leaflet-rails until https://github.com/axyjo/leaflet-rails/pull/66 is resolved
module Leaflet
  require 'rails'
end
```

File: administrate-field-lat_lng.gemspec [hunk 1]
```diff
@@ -18,6 +18,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency 'administrate', '~> 0.2.1'
+  spec.add_dependency 'leaflet-rails', '0.7.7'
```
Comment (pedantic-git) [inline administrate-field-lat_lng.gemspec]: Why is this a hard dependency on 0.7.7 and not the 1.x series? Can we please say `'~> 1.0'` instead?

Comment (klaseskilson) [reply administrate-field-lat_lng.gemspec]: I didn't think too much about it, I just wen't with the version that was used before. Will take a look at `~> 1.0`.

**Comment by klaseskilson** [comment]
@pedantic-git There seem to be an issue (https://github.com/axyjo/leaflet-rails/issues/56) in `leaflet-rails` regarding the marker icon path. Added a fix in fb29f5a643702cb46eed36f2cabc16dec5353be6. 

Should be good to go! Let me know what you think.

Also, FYI: I investigated if the view helpers from `leaflet-rails` could be used, but I couldn't figure out a neat way to make it work since the `lat` and `lng` actually come from two different fields.

**Comment by pedantic-git** [comment]
Thanks very much! I know what you mean about the view helpers - I seem to remember hitting the same issue myself. I'll get a new gem released as soon as these tests pass.

**Comment by klaseskilson** [comment]
Fantastic! Thank you.

**Comment by pedantic-git** [comment]
OK - Travis is misbehaving so I ran the tests locally. Merging now!

**Comment by pedantic-git** [comment]
And released to Rubygems as 1.1.0. Thanks for all your work!

**Comment by klaseskilson** [comment]
Awesome, thanks! No worries. 👍

**Pull request merged**